libriscan/biblios/static/css/index.css

```diff
@@ -0,0 +1,62 @@
+/* --- layout --- */
+.ls-page{max-width:1200px;margin:0 auto;padding:18px}
+.ls-title{font-size:18px;margin:0 0 8px}
+.ls-welcome{font-size:14px;color:#6b7280;margin:0 0 10px}
+.ls-toolbar{display:flex;align-items:center;justify-content:space-between;gap:12px;margin:6px 0 12px}
+.ls-toolbar-left{display:flex;align-items:center;gap:8px}
+.ls-toolbar-right{display:flex;gap:6px;align-items:center}
+.ls-input{height:32px;min-width:340px;padding:0 10px;border:1px solid #e5e7eb;border-radius:6px;font-size:13px}
+.ls-input:focus{outline:none;border-color:#4f46e5}
+.ls-select{height:32px;padding:0 8px;border:1px solid #e5e7eb;border-radius:6px;font-size:13px;background:#fff;cursor:pointer}
+.ls-toggle{display:inline-flex;align-items:center;gap:5px;padding:5px 10px;border:1px solid #e5e7eb;border-radius:6px;background:#fff;cursor:pointer;font-size:13px;color:#374151;transition:all 0.15s}
```

Comment:
These are a lot of custom css for toggle. Please refer to existing toggle filter implemented.
Attempt to use as much of DaisyUI or Tailwind CSS, which will take care of responsiveness too.

https://github.com/Crimson-Vision/Libriscan/blob/stage/libriscan%2Fbiblios%2Ftemplates%2Fbiblios%2Fcomponents%2Fforms%2Fconfidence_toggle.html